src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -737,7 +737,8 @@ private void PrepareAuthorityInstanceForMsal()
             {
                 string b2cAuthority = application.Authority.Replace(
                     new Uri(application.Authority).PathAndQuery,
-                    $"/{ClaimConstants.Tfp}/{_microsoftIdentityOptions.Domain}/{userFlow ?? _microsoftIdentityOptions.DefaultUserFlow}");
+                    $"/{ClaimConstants.Tfp}/{_microsoftIdentityOptions.Domain}/{userFlow ?? _microsoftIdentityOptions.DefaultUserFlow}",
```

Comment:
what is the point of building an authority which is harder to read when debugging / using Fiddler, given that the STS will understand it anyway?

Reply:
I'll have a look.

Reply:
1. The user could configure an authority with a different casing and the replace will fail. 
2. OrdinalIngnoreCase string operations are about 7 times faster than the case sensitive ones